pyramid_openapi3/__init__.py

```diff
@@ -65,7 +65,7 @@ def openapi_view(view: View, info: ViewDeriverInfo) -> t.Optional[View]:
     """
     if info.options.get("openapi"):
 
-        def wrapper_view(context, request):
+        def wrapper_view(context: t.Any, request: Request):
```

Comment:
Probably this is the way:

```
ContextType = t.TypeVar('Context')
View = t.Callable[[ContextType, Request], Response]

def wrapper_view(context: ContextType, request: Request) -> Response:
```
I'll check ...

Reply:
Please submit a PR if it turns out it works. It does look better!